### DISCUSSION TITLE
Issues with renounceOwnership function in chapter 4 challenge

### DISCUSSION THREAD

**Discussion by infinite969** [Cyfrin/foundry-full-course-cu#968] [Q&A]
hey friends, I am getting this notification in my metamask when trying to write the renounceOwnership function of the Chapter 4 challenge on etherscan: "We were not able to estimate gas. There might be an error in the contract and this transaction may fail."

It gives me an option below the notification to proceed with my transaction, but it keeps getting reverted by the EVM when I try and do this. Is this an issue with metamask or am I doing something wrong? I am doing this challenge on the Sepolia network

**Comment by cromewar** [comment]
Hello @infinite969 
Well as we don't want to spoil the challenge solutions I would just say that you are not quite solving the challenge yet.
A hint: try to actually understand the number of decimals and try again.

Hope you solve it soon 😉
